src/main/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurer.java

```diff
@@ -149,7 +149,9 @@ void updateAdminPassword() {
 
             // if ADMIN_PASSWORD is still an empty string, it implies no custom password was provided. We exit the setup.
             if (Strings.isNullOrEmpty(ADMIN_PASSWORD)) {
-                System.out.println("No custom admin password found. Please provide a password.");
+                System.out.println(
+                    "No custom admin password found. Please provide a password via the environment variable OPENSEARCH_INITIAL_ADMIN_PASSWORD."
```

Comment:
Please use the `ConfigConstants.OPENSEARCH_INITIAL_ADMIN_PASSWORD` constant instead of hardcoding the value in the string

Reply:
👍 good idea, I did so in: [163bf49](https://github.com/opensearch-project/security/pull/3879/commits/163bf494a7ebc623a154ccfb2ce587fab346a7b4)